test/staging/Intl402/Temporal/old/date-time-format.js

```diff
@@ -9,7 +9,23 @@ includes: [deepEqual.js]
 features: [Temporal]
 ---*/
 
-//should return an Array"
+// Intl implementations in Node 19 have ICU format changes from earlier Node versions: 
+// - replace some spaces with non-breaking-space unicode characters
+// - change German date separators (these may be bugs)
+const amPmSpace = new Intl.DateTimeFormat("en-US", { timeStyle: "short" }).formatToParts(0).slice(-2)[0].value;
+const dateRangeSeparator = new Intl.DateTimeFormat("en-US", { dateStyle: "short" }).formatRangeToParts(
+  0,
+  1000 * 86400
+)[5].value;
+// May be an Intl bug: the de-AT date separator changed from "." to "/" in Node 19
+const germanDateSeparator = new Intl.DateTimeFormat("de-AT", { dateStyle: "short", calendar: "gregory" }).formatToParts(
+  0
+)[1].value;
+// Probably an Intl bug: the month and year format seems to use a different separator character (in Node 19.2.0)
+// than other date formats.
+const germanMonthYearSeparator = new Intl.DateTimeFormat("de-AT", { year: "numeric", month: "numeric" }).formatToParts(
+  0
+)[1].value;
```

Comment:
Good questions!  I think your changes are good ones, I'll include them in the next push of this PR, hopefully tomorrow.  Also (see discussion below about the German formats) it's likely that much of the German-specific code in this PR will go away, which should further simplify things and shrink the diff.

I agree that it would be good to further reduce the dependency of these tests on CLDR string formats. But I don't think I'll have time to do that work myself. For this PR, I was kinda hoping to do the minimum changes required to get Test262 to pass in both Node 18 and Node 19 so we don't block anyone's testing as folks start upgrading their CIs to v19.

If OK with you, I'd like to leave the "refactor to reduce CLDR format dependency" as an exercise for the future. Perhaps whoever ports these "staging" tests into the mainstream Test262 could also look into refactoring them? Does this sound OK?

Reply:
Yep, done!